app/assets/stylesheets/style.scss

```diff
@@ -10461,7 +10248,7 @@ background: linear-gradient(180deg, rgba(255,252,252,0) 0%, rgba(126,109,109,0.5
 }
 
 .footer {
-  padding: 5rem 0;
+  background-color: black;
```

Comment:
is the magical fix to our white space?

Reply:
I thought it would be but it did nothing. There's a lot in here that we didn't utilize. After much exploring, we essentially built the front end on our own since we didn't utilize bootstrap.